index.html

```diff
@@ -77,35 +83,48 @@ <h6 class="hidden">.</h6>
         <div class="elm_4">.</div>
         <div class="elm_5">.</div>
         <div class="elm_6">.</div>
+        <div class="elm_7">.</div>
+        <div class="elm_8">.</div>
+        <div class="elm_9">.</div>
+        <div class="elm_10">.</div>
+
     </section>
     <section class="work_grid">
     <ul>
         <li class="box">
-            <div class="prev_1 preview item"></div>
-            <div class="title item">
-                <h1>Tonic</h1>
+                <div class="prev_1 preview sub_box"></div>
+                <div class="sub_box_2">
+                <div class="title item">
+                    <h1>Tonic</h1>
+                </div>
+                <ul class="features item">
+                    <li><h4 class="feature_1">CANOPY</h4></li>
+                    <li><div class="dot"></div></li>
+                    <li><h4 class="feature_2">Back End Dev</h4></li>
+                    <li><div class="dot"></div></li>
+                    <li><h4 class="feature_2">2015</h4></li>
+                </ul>
+                <div class="description item">
+                    <p>
+                        A daily selection of privately perzonalized reads; no account or sign-ups required.
+                    </p>
+                </div>
+                <ul class="languages item">
+                    <li><h5>html</h5></li>
+                    <li><h5>css</h5></li>
+                    <li><h5>JavaScript</h5></li>
+                </ul>
+                <div class="button item">
+                    <h3>
+                        <a>See Project</a>
+                    </h3>
+                </div>
             </div>
-            <ul class="features item">
-                <li><h4 class="feature_1">CANOPY</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">Back End Dev</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">2015</h4></li>
-            </ul>
-            <div class="description item">
-                <p>
-                    A daily selection of privately perzonalized reads; no account or sign-ups required.
-                </p>
-            </div>
-            <ul class="languages item">
-                <li><h5>html</h5></li>
-                <li><h5>css</h5></li>
-                <li><h5>JavaScript</h5></li>
-            </ul>
-            <div class="button item"><h3><a>See Project</a></h3></div>
         </li>
-        <li class="box">
-            <div class="prev_2 preview item"></div>
+        
+        <li class="box_rev">
```

Comment:
Since you have to stick to the design as much as possible, kindly observe the images below:
_**Figma Design**_
![original](https://user-images.githubusercontent.com/26856491/121660990-a7452400-ca9b-11eb-9195-238e98cb9478.png)

**_Your Portfolio_** 
![clone](https://user-images.githubusercontent.com/26856491/121661005-aad8ab00-ca9b-11eb-9b07-14318f71cde3.png)

From the images below, the following changes are required:

- [ ] For each of the projects, the width is divided equally: half of it contains the project's picture and the other half contains the project's description Kindly ensure equality of the halves.
- [ ] The `See project` button should be wrapped to fit the text of the button. In your design, the button is taking the entire width of the half.